src/components/SharePopover/SharePopover.tsx

```diff
@@ -103,6 +105,7 @@ export const SharePopover = (props: SharePopoverProps) => {
         renderCopy,
         children,
         onClick,
+        controlAriaLabel,
```

Comment:
`buttonAriaLabel`, we already have button* props

Reply:
ok